Source/Core/Core/ConfigLoaders/IsSettingSaveable.cpp

```diff
@@ -19,133 +19,133 @@ bool IsSettingSaveable(const Config::ConfigLocation& config_location)
+      &Config::GFX_ENHANCE_POST_SHADER.location,
+      &Config::GFX_ENHANCE_FORCE_TRUE_COLOR.location,
+      &Config::GFX_ENHANCE_DISABLE_COPY_FILTER.location,
+      &Config::GFX_ENHANCE_ARBITRARY_MIPMAP_DETECTION.location,
 
       // Graphics.Stereoscopy
 
-      Config::GFX_STEREO_MODE.location,
-      Config::GFX_STEREO_DEPTH.location,
-      Config::GFX_STEREO_CONVERGENCE_PERCENTAGE.location,
-      Config::GFX_STEREO_SWAP_EYES.location,
-      Config::GFX_STEREO_CONVERGENCE.location,
-      Config::GFX_STEREO_EFB_MONO_DEPTH.location,
-      Config::GFX_STEREO_DEPTH_PERCENTAGE.location,
+      &Config::GFX_STEREO_MODE.location,
+      &Config::GFX_STEREO_DEPTH.location,
+      &Config::GFX_STEREO_CONVERGENCE_PERCENTAGE.location,
+      &Config::GFX_STEREO_SWAP_EYES.location,
+      &Config::GFX_STEREO_CONVERGENCE.location,
+      &Config::GFX_STEREO_EFB_MONO_DEPTH.location,
+      &Config::GFX_STEREO_DEPTH_PERCENTAGE.location,
 
       // Graphics.Hacks
 
-      Config::GFX_HACK_EFB_ACCESS_ENABLE.location,
-      Config::GFX_HACK_EFB_DEFER_INVALIDATION.location,
-      Config::GFX_HACK_EFB_ACCESS_TILE_SIZE.location,
-      Config::GFX_HACK_BBOX_ENABLE.location,
-      Config::GFX_HACK_FORCE_PROGRESSIVE.location,
-      Config::GFX_HACK_SKIP_EFB_COPY_TO_RAM.location,
-      Config::GFX_HACK_SKIP_XFB_COPY_TO_RAM.location,
-      Config::GFX_HACK_DISABLE_COPY_TO_VRAM.location,
-      Config::GFX_HACK_DEFER_EFB_COPIES.location,
-      Config::GFX_HACK_IMMEDIATE_XFB.location,
-      Config::GFX_HACK_COPY_EFB_SCALED.location,
-      Config::GFX_HACK_EFB_EMULATE_FORMAT_CHANGES.location,
-      Config::GFX_HACK_VERTEX_ROUDING.location,
+      &Config::GFX_HACK_EFB_ACCESS_ENABLE.location,
+      &Config::GFX_HACK_EFB_DEFER_INVALIDATION.location,
+      &Config::GFX_HACK_EFB_ACCESS_TILE_SIZE.location,
+      &Config::GFX_HACK_BBOX_ENABLE.location,
+      &Config::GFX_HACK_FORCE_PROGRESSIVE.location,
+      &Config::GFX_HACK_SKIP_EFB_COPY_TO_RAM.location,
+      &Config::GFX_HACK_SKIP_XFB_COPY_TO_RAM.location,
+      &Config::GFX_HACK_DISABLE_COPY_TO_VRAM.location,
+      &Config::GFX_HACK_DEFER_EFB_COPIES.location,
+      &Config::GFX_HACK_IMMEDIATE_XFB.location,
+      &Config::GFX_HACK_COPY_EFB_SCALED.location,
+      &Config::GFX_HACK_EFB_EMULATE_FORMAT_CHANGES.location,
+      &Config::GFX_HACK_VERTEX_ROUDING.location,
 
       // Graphics.GameSpecific
 
-      Config::GFX_PERF_QUERIES_ENABLE.location,
+      &Config::GFX_PERF_QUERIES_ENABLE.location,
 
       // UI.General
 
-      Config::MAIN_USE_DISCORD_PRESENCE.location,
+      &Config::MAIN_USE_DISCORD_PRESENCE.location,
 
   };
 
-  return std::find(s_setting_saveable.begin(), s_setting_saveable.end(), config_location) !=
-         s_setting_saveable.end();
+  return std::any_of(s_setting_saveable.cbegin(), s_setting_saveable.cend(),
+    [&, config_location](const Config::ConfigLocation *location) { return *location == config_location; });
```

Comment:
```suggestion
    [config_location](const Config::ConfigLocation *location) { return *location == config_location; });
```

Although I hope this passes a const reference and doesn't make a value type out of `config_location`.

Reply:
I checked this with godbolt.org and if & isn't specified, it *does* get converted to value type.

Reply:
This is fixed in the rebase.

Reply:
This line doesn't show as outdated, though.

Reply:
...looks like I missed one of them. (I fixed the other ones.) Will fix this tonight.